packages/client/components/GenericAuthentication.tsx

```diff
@@ -99,18 +99,24 @@ const GenericAuthentication = (props: Props) => {
           {counterAction}
         </BrandedLink>
       </DialogSubTitle>
-      <GoogleOAuthButtonBlock isCreate={isCreate} invitationToken={invitationToken} />
-      <HorizontalSeparator margin='1rem 0 0' text='or' />
-      <EmailPasswordAuthForm
-        email={email || ''}
-        isSignin={!isCreate}
-        invitationToken={invitationToken}
-        ref={emailRef}
-        goToPage={goToPage}
-      />
+      {window.__ACTION__.AUTH_GOOGLE_ENABLED && <GoogleOAuthButtonBlock isCreate={isCreate} invitationToken={invitationToken} />}
```

Comment:
+1 can you destructure these vars at the top? the long object access makes these hard to read